clusterctl/README.md

```diff
@@ -0,0 +1,96 @@
+# 
+# Deploy a mgmt and a workload cluster with clusterctl in kind
+# 
+# This README is based on, read them before continuing
+https://cluster-api.sigs.k8s.io/clusterctl/overview.html
+https://github.com/Nordix/airship-clusterapi-docs/blob/clusterctl-kashif/clusterCtl-study/testcmd/instrusctions.md
+
+# create a cluster on localhost
+# 
+~$ kind create cluster
+
+# get kubeconfig of new kind cluster
+# 
+~/.cluster-api$ kind get kubeconfig > kubeconfig
+
+# copy files for cluster-template.yaml and metadata.yaml
+# 
+cp ~/work/go/src/sigs.k8s.io/cluster-api-provider-metal3/examples/clusterctl-templates/clusterctl-cluster.yaml ~/.cluster-api/overrides/infrastructure-metal3/v0.3.0/cluster-template.yaml
+
+cp ~/work/go/src/sigs.k8s.io/cluster-api-provider-metal3/metadata.yaml ~/.cluster-api/overrides/infrastructure-metal3/v0.3.0/metadata.yaml
+
+~/.cluster-api$ tree
+.
+├── clusterctl.yaml
```

Comment:
Maybe adding the content of that file or linking to Kashif's PR so that the content of those files is described one way or another

Reply:
link on top of README